typeid/typeid-js/test/valid.ts

```diff
@@ -17,27 +17,27 @@ export default [
   },
   {
     name: "one",
-    typeid: "00000000000000000000000001",
+    typeid: "0000000000e008000000000001",
     prefix: "",
-    uuid: "00000000-0000-0000-0000-000000000001",
+    uuid: "00000000-0000-7000-8000-000000000001",
   },
   {
     name: "ten",
-    typeid: "0000000000000000000000000a",
+    typeid: "0000000000e00900000000000a",
     prefix: "",
-    uuid: "00000000-0000-0000-0000-00000000000a",
+    uuid: "00000000-0000-7000-9000-00000000000a",
   },
   {
     name: "sixteen",
-    typeid: "0000000000000000000000000g",
+    typeid: "0000000000e00a00000000000g",
     prefix: "",
-    uuid: "00000000-0000-0000-0000-000000000010",
+    uuid: "00000000-0000-7000-a000-000000000010",
   },
   {
     name: "thirty-two",
-    typeid: "00000000000000000000000010",
+    typeid: "0000000000e00b000000000010",
     prefix: "",
-    uuid: "00000000-0000-0000-0000-000000000020",
+    uuid: "00000000-0000-7000-b000-000000000020",
```

Comment:
@lukas-runge can these test cases stay the same, or do you need this to be changed? Prefer that all the test cases stay the same just to be doubly sure this is not a breaking change

Reply:
@LucilleH, the test cases do need to be changed because they are not compliant with [RFC 9562](https://datatracker.ietf.org/doc/html/rfc9562#name-uuid-version-7), the proposed standard defining UUIDv7. For a UUIDv7, the version field must always be set to 0b0111 (hex: 7), and the variant field must be 0b10 (hex: 8, 9, A, or B, as the other two bits are randomly assigned). These changes should not cause any breaking issues unless there is incorrect validation of UUIDv7s somewhere, which would be a bug that needs fixing regardless.